spec/strategy_spec.rb

```diff
@@ -163,11 +163,11 @@ def identity
     end
 
     it 'should update user-records with retrieved information' do
-      User.should have(1).record
+      expect(User.count).to eq 1
       User.first.email.should == 'dimitrij@example.com'
     end
   end
-  
+
   describe "POST /users/sign_in (from OpenID provider, success, rememberable)" do
 
     before do
```

Comment:
in this block, at line 175, Devise3.3 saves the user twice, the second time assigning `remember_token`.  In devise3.5 the second save is not called